.github/workflows/publish-package.yml

```
@@ -21,10 +21,23 @@ jobs:
         with:
           node-version: "18.16.0"
           registry-url: "https://registry.npmjs.org"
-      - run: yarn install
-      - run: yarn run build
+      - name: Set version number to environment variable
+        run: node -p -e '`RELEASED_PACKAGE_VERSION=${require("./package.json").version}`' >> $GITHUB_ENV
+      - name: Install node_modules
+        run: yarn install
+      - name: Create package
+        run: yarn run build
         env:
           NODE_ENV: "production"
-      - run: yarn publish --access public
+      - name: Publish package
+        run: yarn publish --access public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Create version tag
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          git tag -a v${{ env.RELEASED_PACKAGE_VERSION }} -m 'Version ${{ env.RELEASED_PACKAGE_VERSION }}'
+          git tag v0 v${{ env.RELEASED_PACKAGE_VERSION }}
```

`-f`オプションが必要な気がします！

```suggestion
          git tag -f v0 v${{ env.RELEASED_PACKAGE_VERSION }}
```

[ref]

- https://git-scm.com/docs/git-tag